core/src/test/java/org/apache/calcite/test/ReflectiveSchemaTest.java

```diff
@@ -625,6 +637,12 @@ private void check(ResultSetMetaData metaData, String columnName,
   private void checkOp(CalciteAssert.AssertThat with, String fn) {
     for (Field field : EveryType.numericFields()) {
       for (Field field2 : EveryType.numericFields()) {
+        //Decimal +-*/ others types has bug see CALCITE-2861
```

Comment:
Two comments here:
1. I suggest to add a space after `//`.
2. Is using the word `arithmetic` better than `+-*/`?

Reply:
Thanks for your suggestion, i will make a change